### PR TITLE
Introduction to Udoo X86 II Ultra

### LINKED ISSUE #2982: Introduction to Udoo x86 II Ultra
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Introduction to Udoo x86 II Ultra 

### Introduction paragraph (2-3 paragraphs):
This article will describe the built in architecture and processing power of  Udoo x86 II Ultra. It will seek to highlight and clarify the different capabilities of Udoo X86 II Ultra and why its way far much better than most Session Border Controllers (SBC). 

This article among others important issues is majorly tailored to explain the advanced features that ensures Ultra X86 II ultra is capable of running software applications that are usually run on a PC on a daily basis.

### Key takeaways: 
1. Understand the best Session Border Controller (SBC) to use?
2. Understand the built in capabilities of Udoo ultra II X86?
3. Understand the decency and wide range of features that Udoo Ultra II x86 presents in order to make an informed decision when acquiring a Session Border Controller (SBC).

### References: 
Not applicable.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by sheldonaswani**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

This fixes #2982

### CODE REVIEW

**Comment by Ericgacoki** [comment]
Greetings @sheldonaswani,
Welcome to Section's Eng-Ed program. We're excited to have you onboard. Thank you for submitting your `first article` for review.
I'll be reviewing it to check and fix any errors before being published!

**Comment by sheldonaswani** [comment]
Greaetings @Ericgacoki 
Thank you. I am happy to work with you.

**Comment by Ericgacoki** [comment]
@sheldonaswani, I have made several changes to improve the flow of your article. Kindly [check them out](https://github.com/section-engineering-education/engineering-education/pull/3178/commits/9e03da2fb346d3a33296b2410d38ef0d747c76dc).

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+Udoo X86 II Ultra session board has the following advantages:
+
+#### Enormous RAM
+Udoo X86 II Ultra comes with an 8GB built in RAM. The advanced version has 4GB of RAM while the basic one has 2GB of RAM. This makes the Udoo X86 II Ultra more powerful than most of the session board controllers on the market.
+
+#### Efficiency
+Udoo X86 II Ultra is capable of running software meant for a normal personal computer. This is made possible by a 2.56GHz quad-core intel CPU and an 8GB RAM.
+
+#### Availability of freely useful guides and community
+Through a helpful community even not all guides are available online.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:61]: I find it hard to understand what this statement means. Could you please clarify it?

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+###	Lower energy consumptions
+The Arduino Leonardo 101 board requires less than 11 Watts of energy which makes it very energy efficient.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:64]: What is the power consumptions compared to other session boards?
Mention why 11 Watts makes Udoo X86 II Ultra efficient.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+#### X86-compatible architecture
+Udoo X86 II Ultra supports most operating systems including Linux, Windows and Android. For Linux and Android, the internal storage can be used as a read only memory with additional SATA `HDD/SSD/eMMC` drive used as /tmp and /home directories.
+
+Udoo II X86 Ultra is also used a gaming emulator as it supports most retro gaming emulators available for a PC. It can also emulate new consoles like play-station II.
+
+#### Decent GPIO support
+The Udoo x86 Ultra comes with 28 `GPIO` ports in total which may seem few, but considering that it has an Arduino 101 embedded on it, this is a decent number.
+
+The Arduino board has an in built functionality for what some pins can be used for (like a six-axis accelerometer and gyroscope) and it adds 12 additional GPIO pins. Several key communication protocols such as the two IIC, two UART, LPC and SDIO are also supported.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:77]: This statement is hard to read. Consider adding more information on the same.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+####  Easy and straightforward set up
+It’s absolutely straightforward to set up. The installation guidelines for the supported operating systems are provided in hard-copy and video form.
+
+#### Arduino compatible
+Udoo x86 II Ultra comes with an embedded Arduino 101, with all upgrades enabled (including 6-axis accelerometer, gyroscope and Bluetooth).
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:83]: You have already mentioned Arduino compatibility in the previous parts. Consider removing this section.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+It also supports a set up for UHD capable media station with a wide variety of applications like Spotify, Netflix, YouTube, Kodi etc.
+
+### Cons
+Udoo X86 II Ultra might not the best choice for the following reasons:
+
+#### Expensive
+Udoo x86 II Ultra is relatively expensive compared to other session board controllers on the market.
+
+#### Lack of official documentation
+Apart from the hardware specifications sheet, there is no official documentation publicly available. Also, there is generally not too many project examples or information available on the official website for Udoo x86 II ultra at the moment. Therefore, users have a hard time finding out how to use the board.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:97]: Provide a link to the official site mentioned above.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+Through a helpful community even not all guides are available online.
+
+###	Lower energy consumptions
+The Arduino Leonardo 101 board requires less than 11 Watts of energy which makes it very energy efficient.
+
+#### Large storage capacity
+Udoo X86 II Ultra box comes with a 32GB embedded multimedia card. However, users might decide to upgrade the storage to fit their desire. Though expensive, an SSD card is usually the best option. Additionally, a micro-SD card can also be used as a storage media.
+
+#### X86-compatible architecture
+Udoo X86 II Ultra supports most operating systems including Linux, Windows and Android. For Linux and Android, the internal storage can be used as a read only memory with additional SATA `HDD/SSD/eMMC` drive used as /tmp and /home directories.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:70]: I've noted a few acronyms that might be unfamiliar to beginners. Provide the full names and a short description of relevant abbreviations.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]
Hello @sheldonaswani, please work on the suggestions attached below before we proceed. Ping me when you're done.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 2 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:64]: Already Addressed.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 6 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:70]: Already Addressed.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 4 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:83]: Already removed.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 5 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:97]: Already addressed.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 6 above)
Comment (Ericgacoki) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:70]: I can't see any updates yet! Did you forget to commit them?

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 1 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:61]: Already addressed.

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md (hunk 3 above)
Comment (sheldonaswani) [reply content/articles/introduction-to-udoo-x86-ii-ultra/index.md:77]: Already addressed.

**Comment by Ericgacoki** [comment]
@sheldonaswani, push the changes you've made. You say you've addressed the suggestions but there's no commit to verify that.

**Comment by sheldonaswani** [comment]
@Ericgacoki Kindly check i have already committed.

**Review by Ericgacoki — APPROVED** [review]

**Comment by hectorkambow** [comment]
@sheldonaswani 
Upon a quick glance this article seems rather short on substance. 

It seems rather light. 

cc @Ericgacoki

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by Ericgacoki** [comment]
Hey @sheldonaswani,
Upon further discussion with other reviewers, it has come to our attention that your article does not meet the required standards to be published. Simply because this article has been covered in detail by various sites, it's relatively short and conveys quite minimal information.

According to our guidelines:
"We lean towards content that offers an original viewpoint/approach around programming-oriented topics."

What's next?
1. You can go ahead and build a viable project based on `UDOO X86 II Ultra` and the embedded Arduino board.
2. If (1) seems difficult to you or for some reason, you can't be able to do so at the moment, we'll permanently close this PR and ask you to suggest another topic.

@sheldonaswani, Please let us know what you think (based on the above options). Thank you!

cc: @hectorkambow

**Comment by sheldonaswani** [comment]
Good evening @Ericgacoki 
I can change  the topic to ###Udoo x86 Ultra compatibility with arduino

Let me know if this is right?

cc: @hectorkambow

**Comment by Ericgacoki** [comment]
How will that add value to this article?

**Comment by Ericgacoki** [comment]
Does changing the topic mean you'll change the whole content as well?

File: content/articles/introduction-to-udoo-x86-ii-ultra/index.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+- [Introduction to Udoo X86 II Ultra](#introduction-to-udoo-x86-ii-ultra)
+- [Pros](#pros)
+- [Cons](#cons)
+- [Conclusion](#conclusion)
+
+### Prerequisites
+To follow through this article, you need to have a basic understanding of `session board controllers`
+
+### Introduction to Udoo X86 II Ultra
+Udoo X86 II Ultra session board controller's power is comparable to that of a new typical personal computer in the market today. Udoo X86 II Ultra comes embedded on the same platform with an Arduino Leonardo 101 board made with a more powerful X86 maker architecture.
```
Comment (Ericgacoki) [inline content/articles/introduction-to-udoo-x86-ii-ultra/index.md:29]: Why would you talk about compatibility with Arduino when it has it embedded already?

**Comment by sheldonaswani** [comment]
I am changing my topic.

**Pull request closed without merging**